lib/commands/init.js

```diff
@@ -141,8 +101,20 @@ module.exports = Command.extend({
       })
       .then(() => {
         if (commandOptions.skipGit === false) {
-          return gitInit.run(commandOptions, rawArgs);
+          return this.runTask('GitInit', commandOptions, rawArgs);
         }
       });
   },
+
+  _env(name) {
+    if (name === 'GenerateFromBlueprint') {
```

Comment:
i had it in there to preserve semantics (we were not previously passing `testing` to this task, but you've convinced me to just always pass those options